src/ray/object_manager/plasma/create_request_queue.cc

```diff
@@ -63,6 +63,12 @@ std::pair<PlasmaObject, PlasmaError> CreateRequestQueue::TryRequestImmediately(
     const CreateObjectCallback &create_callback, size_t object_size) {
   PlasmaObject result = {};
 
+  // Immediately fulfill it using the fallback allocator.
+  if (RayConfig::instance().plasma_unlimited()) {
```

Comment:
Shouldn't we try to go through the normal creation path first? Otherwise we will end up using the fallback allocator before trying to evict objects, etc.

Reply:
CreateObject() does trigger eviction, etc.; the only difference is we don't block for oom_grace_period_s prior to fallback.

Reply:
Okay, I misunderstood the comment. Can you clarify it? It makes it sound like the object will immediately get allocated with the filesystem.